src/main/java/org/zalando/nakadi/controller/EventPublishingController.java

```diff
@@ -100,19 +106,23 @@ private ResponseEntity postEventInternal(final String eventTypeName,
         }
     }
 
+    private void reportSLOs(final long startingNanos, final int totalSizeBytes, final int eventCount) {
+        final long msSpent = TimeUnit.NANOSECONDS.toMillis(System.nanoTime() - startingNanos);
```

Comment:
I will rephrase the question: "why do you convert from ns to ms, just use ms right away" :)